tests/main/layout-symlink-bind-revert/task.yaml

```diff
@@ -17,7 +21,7 @@ execute: |
 
     echo "The application is refreshed with another layout"
     snap install --dangerous ./app_2_all.snap
-    app 2>&1 | MATCH '/snap/app/x2/bin/app: 2: /snap/app/x2/bin/app: /opt/runtime/runner: not found'
+    app 2>&1 | MATCH "RUNTIME: Hello from the app"
```

Comment:
Yeah, I was surprised that we had a regression test :) This test used to fail because of missing propagation between the layout and the content interface used internally.